src/lib/y2firewall/clients/installation_finish.rb

```diff
@@ -73,6 +73,9 @@ def configure_firewall
       # Convenience method to enable / disable the firewalld service depending
       # on the proposal settings
       def configure_firewall_service
+        # do not run in autoyast as it is done in second stage (bsc#1177778)
+        return if Yast::Mode.auto
```

Comment:
Would be nice to configure according to the profile in case of defined section but that is or should be already done in SP3 as it is already configured during the first stage. Thus, for the time being it is ok and nice for SP2.